packages/loader/container-loader/src/contracts.ts

```diff
@@ -197,20 +207,22 @@ export interface IConnectionManagerFactoryArgs {
 }
 
 /**
- *
+ * Gets the name of the Fluid package.
  * @param codeDetails- - Data structure used to describe the code to load on the Fluid document
- * @returns The name of the Fluid package
  */
 export const getPackageName = (
 	codeDetails: IFluidCodeDetails | undefined,
 ): IContainerPackageInfo => {
-	let containerPackageName;
+	// TODO: use a real type
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	let containerPackageName: any;
```

Comment:
Not for this PR but feels like there's a bug here... the first check checks if `codeDetails` has a `name` property... and if it does, it uses not that but _the whole_ `codeDetails` as if it was a string with the package name, and returns it down below? Should it be `return codeDetails.name` in the first if block? @sonalideshpandemsft / @sonalivdeshpande I see you were the last to touch this bit, does it ring a bell? Or do you know who might be able to tell for sure?

Reply:
Yeah, I was wondering about this one as well. Definitely seems wrong.

Reply:
maybe @ChumpChief?

Reply:
Agree, looks like this was intended to mirror [this](https://github.com/microsoft/FluidFramework/commit/6783a8ecc5a3e15c7a134b868e99d635765b443a#diff-17134f977a235c274ffbfef6f7aca4d273656e9eec21d187bce629968258ac5eR171-R178) but just missed a little bit.

Reply:
https://github.com/microsoft/FluidFramework/pull/21849